src/views/Onboarding/OnboardingPassword.vue

```diff
@@ -82,11 +82,7 @@ export default {
   computed: {
     passwordMatch() {
       if (!this.password || !this.confirmPassword) return false
-      if (
-        this.password.length === this.confirmPassword.length &&
-        this.password !== this.confirmPassword
-      )
-        return true
+      if (this.password !== this.confirmPassword) return true
```

Comment:
return true? If the passwords _dont_ equal. Shouldn't `passwordMatch` return false?

its also generally better to do things in this order:
```
if (...) return x
if (...) return x

return y
```
Where x or y can be true or false

Reply:
Fixed.